lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -16,6 +16,122 @@ module.exports = [
       }, {
         id: 'wordpress',
       }],
+      TagsBlockingFirstPaint: [
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dobetterweb/dbw_tester.css?delay=100',
+          },
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/unknown404.css?delay=200',
+          },
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=2200',
+          },
+
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dbw_partial_a.html?delay=200',
+          },
+        },
+        {
+          tag: {
+            tagName: 'LINK',
+            url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped',
+            mediaChanges: [
```

Comment:
I want these media assertions here, because this entire tags gatherer is susceptible to change w/ #2065.